src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -155,6 +155,25 @@ template<class T> void DeleteDbiMemory(T *p)
     g_pAllocator->Free((BYTE*) p);
 }
 
+// Delete memory and invoke dtor for memory allocated with 'operator (forDbi) new[]'
+// There's an inherent risk here - where each element's destructor will get called within
+// the context of the DAC. If the destructor tries to use the CRT allocator logic expecting
+// to hit the DBI's, we could be in trouble. Those objects need to use an export allocator like this.
+template<class T> void DeleteDbiArrayMemory(T *p, int len)
```

Comment:
A tiny nit - maybe using "count" instead of "len" would make it clearer that it is the number of elements and not the size of the underlying memory.